docs/resources/volume.md

```diff
@@ -4,9 +4,9 @@ subcategory: "Unity Catalog"
 # databricks_volume (Resource)
 Volumes are a new Unity Catalog (UC) capability for accessing, storing, governing, and organizing files. Volumes unlock new processing capabilities for data governed by the Unity Catalog, including support for most machine learning and data science workloads. You can use volumes to store and access files in any format; data can be structured, semi-structured, or unstructured.
 
-With Volumes, files managed centrally in Unity Catalogare are organized under a 3-level namespace: `<catalog>.<schema>.<volume>`.
+Volumes are organized under a 3-level namespace: `<catalog>.<schema>.<volume>`.
 
-This resource does Creates, Read, Update, Delete and List Unity Catalog volumes.
+This resource supports Create, Get, Update, Delete and List Unity Catalog Volumes.
```

Comment:
Technically speaking, it's just Create/Update/Delete - Get is implied

Reply:
```suggestion
This resource manages Volumes in Unity Catalog.
```